src/Components/Alert/Components/Modal.tsx

```diff
@@ -32,32 +31,20 @@ export const Modal: React.FC<AlertModalProps> = ({
           : { backgroundColor: "transparent" }
       }
       dialogProps={{
-        width: ["100%", 900],
-        height: ["100%", "90%"],
-        maxHeight: [null, 800],
+        height: ["100%", "auto"],
+        width: ["100%", "auto"],
+        maxHeight: ["auto", "90%"],
+        minWidth: ["auto", 500],
       }}
       {...modalProps}
       title="Create Alert"
     >
       <Box
         width="100%"
+        height="100%"
         position="relative"
-        bg="white100"
         overflowY="auto"
-        style={{
-          WebkitOverflowScrolling: "touch",
-          boxShadow: DROP_SHADOW,
-          ...(isMounted
-            ? {
-                opacity: 1,
-                transform: "translateX(0)",
-                transition: "opacity 5000ms, transform 5500ms",
-              }
-            : {
-                opacity: 0,
-                transform: "translateX(-2000px)",
-              }),
-        }}
```

Comment:
fyi — here's the behavior I'm seeing on mWeb. Think it's best to disable for now and we can sync up with Barney re: transitions.


https://github.com/artsy/force/assets/123595/70b66a4e-0561-4854-b9ab-9c66f41be4e4